qemu/tests/qemu_io_blkdebug.py

```diff
@@ -94,8 +95,8 @@ def run(test, params, env):
             try:
                 image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
                 output = ""
-            except error.CmdError, err:
-                output = err.result_obj.stderr
+            except process.CmdError, err:
```

Comment:
When the command executed in process.run() exit, need to catch this exception for verify the result of qemu_io_blkdebug case.